palantirnet/search_api_federated_solr
# UMUS-128 Pull new version (D7)

This uses the work from palantirnet/federated-search-react#4 to pull the dependency from the new private repo.

To test:

- Checkout this branch at src/search_api_federated_solr-7.x
- `drush @uofmhealth.local cc all`
- visit https://www.uofmhealth.local/search-app?search=keith and see that the app works
